src/slate-editor/slate-editor.stories.tsx

```diff
@@ -16,6 +16,19 @@ const hotkeyMap: HotkeyMap = {
 const baseValue = "This editor-only example has no toolbar but keyboard shortcuts should work, ";
 const initialValueWithHistory = textToSlate(baseValue + "including mod+[shift]+z for undo/redo.");
 
+export const ReadOnly = () => {
```

Comment:
This was an unrelated test case I'd been working on previously. Seemed like it made sense to get it into this release.